### ISSUE TITLE
Allow Veteran to File a Mileage-Only Claim for a Past Appointment

### ISSUE THREAD

**Issue by loripusey** [department-of-veterans-affairs/va.gov-team#68926]
Allow Veterans to file a mileage-only claim for a Past Appointment. For example, receive an SMS the day after an appointment, request a LoROTA entry by replying, then receive a link back to file a claim for that visit. Veterans would have access to see their past appointments from the last 30 days, but only be able to file a reimbursement for the appt information provided from the VeText payload. 

In short, allow Veterans to file a mileage-only claim for a 1 or more past appointments on the same day at the same facility. 

Assumes BTS3 API lag response time has been resolved. E.g., no need for f/u SMS messages.

**Comment by benbrasso-agile6** [comment]
@loripusey I think I'm going to build out some design tickets for this Epic. I mean, it's possible we never do the engineering work, but I'm not sure "what's next" for design. This could be something we chip away at, while engineers work on their priorities.
